chord_metadata_service/phenopackets/admin.py

```diff
@@ -12,26 +12,11 @@ class PhenotypicFeatureAdmin(admin.ModelAdmin):
     pass
 
 
-@admin.register(m.Procedure)
-class ProcedureAdmin(admin.ModelAdmin):
-    pass
-
-
-@admin.register(m.HtsFile)
-class HtsFileAdmin(admin.ModelAdmin):
-    pass
-
-
 @admin.register(m.Gene)
 class GeneAdmin(admin.ModelAdmin):
```

Comment:
should this go away? what's happening with the gene model?

Reply:
Removed the gene model and associated code.